service/common/src/main/java/org/apache/polaris/service/ratelimiter/RateLimiterFilter.java

```diff
@@ -56,7 +58,11 @@ public void filter(ContainerRequestContext ctx) throws IOException {
     if (!rateLimiter.canProceed()) {
       polarisEventListener.onBeforeRequestRateLimited(
           new BeforeRequestRateLimitedEvent(
-              ctx.getMethod(), ctx.getUriInfo().getAbsolutePath().toString()));
+              PolarisEvent.createEventId(),
+              ctx.getMethod(),
+              ctx.getUriInfo().getAbsolutePath().toString()),
+          CallContext.getCurrentContext(),
+          ctx.getSecurityContext());
       ctx.abortWith(Response.status(Response.Status.TOO_MANY_REQUESTS).build());
```

Comment:
This is quite a dangerous call site. An attacker can easily overload the system when rate-limiting kicks in.
I consider this a legit security issue that would justify a CVE.